src/alchemlyb/preprocessing/subsampling.py

```diff
@@ -102,101 +290,208 @@ def statistical_inefficiency(df, series=None, lower=None, upper=None, step=None,
     decreases a false sense of accuracy and is deemed the more careful and
     conservative approach.
 
+    References
+    ----------
+    [1] J. D. Chodera, W. C. Swope, J. W. Pitera, C. Seok, and K. A. Dill. Use of the weighted
```

Comment:
I would use reST citations and have one central citations page. At the end of the day, I think users are more likely to read the online docs for this kind of information than when they read the raw doc strings.